department-of-veterans-affairs/caseflow
# Wmedders21/appeals 61467

Resolves [APPEALS-61467](https://jira.devops.va.gov/browse/APPEALS-61467)

# Description
Please explain the changes you made here.

Updates the TaskRows component to be able to handle a formatted array of instructions for ReviewTranscriptTask to display case timeline details. 

## Acceptance Criteria

## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below
- In Rails console, find an appeal and set it to variable `appeal`
  - i.e. `appeal = Appeal.all.limit(20).last`
- Create a ReviewTranscriptTask for that appeal and set to variable `task`
```
task = ReviewTranscriptTask.create!(  
   appeal: appeal,    
   parent: appeal.root_task,    
   assigned_to: User.find_by(css_id: "BVASYELLOW"),    
   assigned_by: User.system_user,
   instructions: [COPY::REVIEW_TRANSCRIPT_TASK_DEFAULT_INSTRUCTIONS, COPY::UPLOAD_TRANSCRIPTION_VBMS_CANCEL_ACTION_TYPE, "these are a bunch of notes"]
)
``` 
- Authenticate a user
  - `RequestStore[:current_user] = User.find_by(css_id: "BVASYELLOW")`
- Cancel the task
  - `task.cancel_task_and_child_subtasks`
-  Copy the appeal uuid
  - `appeal.uuid`
- In a browser, log in to Caseflow as BVASYELLOW
- Navigate to `/queue/appeals/<appeal_uuid>`
- Scroll down to the case timeline
- Expect to see an entry for ReviewTranscriptTask as described in [APPEALS-61467](https://jira.devops.va.gov/browse/APPEALS-61467)

# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
